src/Moka/Proxy/ProxyInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+declare(strict_types=1);
+
+namespace Moka\Proxy;
+
+use PHPUnit_Framework_MockObject_MockObject as MockObject;
+
+/**
+ * Interface ProxyInterface
+ * @package Moka\Proxy
+ */
+interface ProxyInterface
```

Comment:
What's the reason behind this interface? What purpose of replacement or multiple inheritance do you think will be solvable with this?